File: lv1/로그인성공.py
def solution(id_pw, db):
    isloged = False
    for i in db:
        if i[0] == id_pw[0]:
            if i[1] == id_pw[1]:
                answer = 'login'
            else:
                answer = 'wrong pw'
            isloged = True
            break
    if isloged == False:
        answer = 'fail'
    return answer

File: lv1/로그인성공.js
function solution(id_pw, db) {
  var answer = '';
  db.some(([id, pw], i) => {
    if (id_pw[0] === id && id_pw[1] === pw) {
      answer = "login";
      return true;
    }
    if (id_pw[0] === id) {
      answer = "wrong pw";
    }
    if (i + 1 === db.length && answer === '') {
      answer = "fail";
    }
  })
  return answer;
}